packages/web/src/components/audio-transaction-icon/AudioTransactionIcon.module.css

```diff
@@ -0,0 +1,64 @@
+.transactionIcon {
+  display: flex;
+  position: relative;
+  padding: 12px;
+}
+.transactionIconMini {
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 6px;
+  left: 6px;
+  width: 18px;
+  height: 18px;
+  box-shadow: 0 0 0 2px var(--static-white);
+  border-radius: 100%;
+}
+.transactionIconMini.stripe {
+  background-color: #635bff;
+}
+.transactionIconMini.coinbase {
+  background-color: #0052ff;
+}
+.transactionIconMini.sent {
+  padding: 2px;
```

Comment:
Dang, looks like it wasn't as simple of a drop-in. In theory it should be aligned via the flexbox?

Reply:
this is to match size bc the icon is a little too big. was shorter than setting the height and width, but i can see how this can cause confusion

Reply:
Makes sense, was just curious more than anything!